### DISCUSSION TITLE
米游社

### DISCUSSION THREAD

**Discussion by Yingl-1** [Adpro-Team/GKD_subscription#95] [问答] [answered]
米游社的原神和星铁自动签到出错，不会退出，会进入当前游戏的主页

**Comment by adproqwq** [comment]
需要快照

**Reply by Yingl-1** [reply]
https://i.gkd.li/i/14967627
//原神没反应
https://i.gkd.li/i/14967631
https://i.gkd.li/i/14967633
//星铁反应了，不过这次没进主页，只不过看第二个链接一直在gkd，所以按理说是要进主页的，我的是，但是不知道为什么没进，一直在gkd

**Reply by adproqwq** [reply, marked as answer]
15095b2

**Reply by Yingl-1** [reply]
希望可以加个幕布的
https://i.gkd.li/i/15066979

**Reply by adproqwq** [reply]
c6250c4

**Reply by Yingl-1** [reply]
https://i.gkd.li/i/15223416
番茄阅读

**Reply by adproqwq** [reply]
ccecd5f
